xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-rest/src/main/java/org/xwiki/localization/rest/LocalizationSource.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.localization.rest;
+
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+import org.xwiki.stability.Unstable;
+
+/**
+ * Exposes the wiki translations through REST.
+ *
+ * @version $Id$
+ * @since 13.3RC1
+ */
+@Path("/wikis/{wikiName}/localization/translations")
+@Unstable
+public interface LocalizationSource
```

Comment:
Doesn't match the URL and we use "**Re**source" everywhere else. So it should be ``TranslationsResource``. The ``LocalizationResource`` will be used for ``/wikis/{wikiName}/localization`` (i.e. is the parent resource). @tmortagne suggested to have two separate resources for ``localization`` and ``translations`` because localization is much more than translations and we wanted to keep it open to add more later. For the scope of this PR you only need to implement the translations resource.

Reply:
Would be nicer for reusability to have this API (and the model) in an API module which does not force you to depend on oldcore (which is the case of any implementation of XWikiRestComponent right now).

Reply:
Would it be ok to move this class to `xwiki-platform-localization-api` in this case?

Reply:
I'm not sure, it would force a dependency on ``javax.ws.rs`` for ``localization-api``.

Reply:
Yeah I'm not a fan either of the extra dependencies to add to xwiki-platform-localization-api.

Reply:
Or should I two sub modules `xwiki-platform-localization-rest-api` and `xwiki-platform-localization-rest-default`?

Reply:
I'm working on this part.